src/WebSdk/Web/Sdk/Sdk.props

```diff
@@ -23,19 +23,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
   </PropertyGroup>
 
-  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" Condition="'$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' != 'browser'" />
```

Comment:
Do you want to store the value of the condition in a property so that the condition check in each of these will look simpler `[MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' != 'browser'`?